### DISCUSSION TITLE
I love achievements!

### DISCUSSION THREAD

**Discussion by ATreeShine** [community/community#80779] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I like the achievements very much. But I would like to know I don't know how to get all the achievements available.

**Comment by Cfomodz** [comment]
The achievements are available as you complete common actions within the ecosystem. You can learn more here: https://github.com/Schweinepriester/github-profile-achievements

They can be turned on/off in your profile https://github.com/settings/profile#profile-settings-heading
